web/src/features/fbaCalculator/components/map/FBAMap.tsx

```diff
@@ -18,14 +17,16 @@ import { makeStyles } from '@material-ui/core/styles'
 import { ErrorBoundary } from 'components'
 import { selectFireWeatherStations } from 'app/rootReducer'
 import { source } from 'features/fireWeather/components/maps/constants'
+import Tile from 'ol/layer/Tile'
+import TileWMS from 'ol/source/TileWMS'
 
 export const fbaMapContext = React.createContext<ol.Map | null>(null)
 
-const zoom = 6
+const zoom = 5.45
+const BC_CENTER_FIRE_CENTERS = [-124.16748046874999, 54.584796743678744]
```

Comment:
what is this for? It seems like the name of the variable doesn't match the geographic coord. Is it just the centre of BC?

Reply:
Yup, existing center assumes a different zoom level.

Reply:
We can probably switch this to querying the extent from the server - but I'd hold off on that for now.